### ISSUE TITLE
Facial Recognition section "Context" image preview shows as native resolution

### ISSUE THREAD

**Issue by AzAel76** [LibrePhotos/librephotos#503]
## 🐛 Bug Report

* [ ] I am including my log files in zip format.
* [ ] I have looked for similar issues among all the former issues including closed ones.

### What Operating system and version is LibrePhotos running on:
Unraid 6.10

### What architecture is LibrePhotos running on:
x64

### How is LibrePhotos installed:
Docker

### If running via Docker or Kubernetes please list version including docker-compose:
Latest

### Are you running LibrePhotos on a virtual machine if so please list:
NA

### How is you picture library mounted on the host (or in the virtual machine):
Native Docker volume and Nextcloud

### Description of issue:
When clicking on the 'Orignal Image' button on the bottom right of the face thumbs, the pop out displays the original image at full resolution and always from the bottom left of the image. 

Basically means can only view the bottom left of any image that exceeds the page view size.

### How can we reproduce it:
Follow issue description.

#### Additional Information:
Tested in Vivialdi, Chrpome, Edge and Firefox

* Include a ZIP file containing your `log` files.
* Provide pictures or links to videos that clearly demonstrate the issue.

**Comment by derneuere** [comment]
I think this should be mostly fixed by this week's release. A user reported, that the window sometimes spawns in the wrong location, which I should investigate further.

**Comment by AzAel76** [comment]
Yes, always seems to anchor the spawned viewport at the click location.and click location is always the bottom left.

**Comment by AzAel76** [comment]
Yes. Latest Release "Mostly" fixes it. Still seems to have issues with images have a portrait orientation. looks like you may have set a max width variable as width of the portrait orientation images seem to be he same width as the landscape orinetation previews

**Comment by derneuere** [comment]
I think that this is fixed now :)
